template/src/project_name/main/settings.py

```diff
@@ -144,3 +146,18 @@
         },
     },
 }
+
+# Sentry
+sentry_sdk.init(
+    dsn=env("SENTRY_DSN", default=None),
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for tracing.
+    traces_sample_rate=env.float("SENTRY_TRACES_SAMPLE_RATE", default=1.0),
+    # Set profiles_sample_rate to 1.0 to profile 100%
+    # of sampled transactions.
+    # We recommend adjusting this value in production.
+    profiles_sample_rate=env.float("SENTRY_PROFILES_SAMPLE_RATE", default=1.0),
+    integrations=[
+        DjangoIntegration(),
+    ],
```

Comment:
issue: we want to explicitly set the environment to the value of the `SENTRY_ENV` with `production` as the default:

```suggestion
    ],
    environment=os.environ.get("SENTRY_ENV", "production"),
```

Note that by default Sentry does read this from `SENTRY_ENVIRONMENT` - feel free to make this fallback to that before `production`, but right now we're using `SENTRY_ENV` across all of our apps and imo Sentry doesn't make this env variable as obvious as I'd like so I think it's a bit nicer for us to be making it more obvious this is an option + what the default is

Reply:
Happy to add this - but please us `env("SENTRY_ENV", "production")` rather than using `os.environ` directly.

Reply:
Whoops yeah good catch - I forgot to update that when copying across the snippet from an existing app (which we should totally move to using the `environ` package at some point!); I've updated to use `env` instead